jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/model/dto/ProjectToChannelConfigurationDTO.java

```diff
@@ -69,7 +68,7 @@ public String getProjectKey() {
     }
 
     @SuppressWarnings("unused")
-    public List<String> getOrderedChannelIds() {
+    public List<ConversationKey> getOrderedChannelIds() {
```

Comment:
After changing the type list contains `getOrderedChannelIds()` method name becomes misleading. Could you rename it to `getOrderedConversationKeys()`? It may be done in another PR.